dockstore-client/Hello.java

```diff
@@ -0,0 +1 @@
+public class Hello {}
```

Comment:
What's this used for?

Reply:
Looks like a mistake, might be generated during integration tests and I accidentally added it